4_advanced_usage/75_relay_servers.txt

```diff
@@ -25,134 +25,22 @@ The procedure will:
-do
-        /usr/bin/htpasswd -bc ${i} rudder rudder
-done
-
-touch /opt/rudder/etc/rudder-networks.conf
-
-----
-
-===== Please create the appropriate file for your OS, and copy the following content in it
-
-- /etc/httpd/conf.d/rudder-default.conf file on RHEL-like
-- /etc/apache2/sites-enabled/rudder-default file on Debian-like
-- /etc/apache2/vhosts.d/rudder-default.conf file on SuSE
-
-----
-<VirtualHost *:80>
-        ServerAdmin webmaster@localhost
-        # Expose the server UUID through http
-        Alias /uuid /opt/rudder/etc/uuid.hive
-        <Directory /opt/rudder/etc>
-                Order deny,allow
-                Allow from all
-        </Directory>
-        # WebDAV share to receive inventories
-        Alias /inventories /var/rudder/inventories/incoming
-        <Directory /var/rudder/inventories/incoming>
-                DAV on
-                AuthName "WebDAV Storage"
-                AuthType Basic
-                AuthUserFile /opt/rudder/etc/htpasswd-webdav-initial
-                Require valid-user
-                Order deny,allow
-                # This file is automatically generated according to
-                # the hosts allowed by rudder.
-                Include /opt/rudder/etc/rudder-networks.conf
-                <LimitExcept PUT>
-                        Order allow,deny
-                        Deny from all
-                </LimitExcept>
-        </Directory>
-        # WebDAV share to receive inventories
-        Alias /inventory-updates /var/rudder/inventories/accepted-nodes-updates
-        <Directory /var/rudder/inventories/accepted-nodes-updates>
-                DAV on
-                AuthName "WebDAV Storage"
-                AuthType Basic
-                AuthUserFile /opt/rudder/etc/htpasswd-webdav
-                Require valid-user
-                Order deny,allow
-                # This file is automatically generated according to
-                # the hosts allowed by rudder.
-                Include /opt/rudder/etc/rudder-networks.conf
-                <LimitExcept PUT>
-                        Order allow,deny
-                        Deny from all
-                </LimitExcept>
-        </Directory>
-        # Logs
-        ErrorLog /var/log/rudder/apache2/error.log
-        LogLevel warn
-        CustomLog /var/log/rudder/apache2/access.log combined
-
-</VirtualHost>
-----
-
-Finally, restart Apache to take these changes into account:
-
-----
-
-# RHEL-like (RHEL, CentOS, Scientific)
-service httpd restart
-
-# Debian-like (Debian, Ubuntu) and SuSE
-service apache2 restart
-
-----
-
-
 ==== Installation procedure - On the root server
 
 ===== Set the node as a Rudder relay
 
 Launch the rudder-node-to-relay script on the root server, supplying the UUID of the
-host to be added as a relay to it:
+host to be added as a relay to it. You can use the command 'rudder agent info'
+to get the UUID of your node.
```

Comment:
You could improve this with a PR on rudder-doc on the 3.1 branch though, it would indeed be much simpler!